mailet/api/src/main/java/org/apache/mailet/MailetConfig.java

```diff
@@ -38,17 +38,19 @@
      * or null if the parameter does not exist.
      *
      * @param name the name of the initialization parameter
+     * @param clazz the type of the returned value
      * @return the value of the initialization parameter, or null
+     * @throws ClassCastException
      */
-    String getInitParameter(String name);
```

Comment:
IMO we should keep the `String getInitParameter(String name);` method as it is more convenient most of the time...